fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -957,6 +964,18 @@ void DDQueue::launchQueuedWork(RelocateData launchData, const DDEnabledState* dd
 	launchQueuedWork(combined, ddEnabledState);
 }
 
+DataMoveType newDataMoveType() {
+	DataMoveType type = DataMoveType::LOGICAL;
+	if (deterministicRandom()->random01() < SERVER_KNOBS->DD_PHYSICAL_SHARD_MOVE_PROBABILITY) {
+		type = DataMoveType::PHYSICAL;
+	}
+	if (type != DataMoveType::PHYSICAL && SERVER_KNOBS->ENABLE_PHYSICAL_SHARD_MOVE_EXPERIMENT) {
+		type = DataMoveType::PHYSICAL_EXP;
```

Comment:
Why we set `type` be set as `PHYSICAL_EXP` when `type != DataMoveType::PHYSICAL`. What is the purpose of  `PHYSICAL_EXP`?

Reply:
The goal is to enable physical shard move for only TSS. When `type == DataMoveType::PHYSICAL`, both TSS and SS will do physical shard move.

Reply:
I see. Thanks!